src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -502,11 +502,7 @@ public void CleanupAfterEachTest()
 
             string actualUpdatedConnectionString = updatedRuntimeConfig.DataSource.ConnectionString;
 
-            // SourceLink appends the commit ID to the assembly metadata. The application name is extracted from the
-            // assembly metadata. So, the connection string after appending the application name will be of the form
-            // dab_oss_1.0.0+<commit-id> or dab_hosted_1.0.0+<commit-id> depending on oss or hosted scenario respectively.
-            // So, the updated connection string is validated to check if it starts with dab_oss_1.0.0 or dab_hosted_1.0.0.
-            Assert.IsTrue(actualUpdatedConnectionString.StartsWith(expectedUpdatedConnectionString));
+            Assert.IsTrue(actualUpdatedConnectionString == expectedUpdatedConnectionString);
```

Comment:
```suggestion
            Assert.AreEqual(actualUpdatedConnectionString, expectedUpdatedConnectionString);
```

Using Equal is better for troubleshooting because in case of failures, it will print out both the string in the logs so we would know immediately why the test could have failed.